sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -892,20 +896,50 @@ static DeployGate getInstance() {
         return sInstance;
     }
 
-    void sendCrashReport(Throwable ex) {
+    void sendCrashReport(/* non-null */ Throwable ex) {
         if (mRemoteService == null)
             return;
         
         Bundle extras = new Bundle();
-        extras.putSerializable(DeployGateEvent.EXTRA_EXCEPTION, ex);
         try {
+            if (getDeployGateVersionCode() > 41 /* greater than ver 1.7.2 */) {
```

Comment:
41 is the current recent version of DeployGate client app.

Reply:
It should be a named constant than a magic number, like:

```
static final int EXCEPTION_ROOT_CAUSE_SUPPORT_VERSION = 41;
```

Reply:
Addressed in https://github.com/DeployGate/deploygate-android-sdk/pull/14/commits/9ea3e03a209116d098c035cdfb49d40ae9d481da